### PR TITLE
12 tabbing

### PR DESCRIPTION
**Pull request by pontuslaestadius**

### CODE REVIEW

File: Server/src/compile.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Author: Pontus Laestadius
+for f in *.erl; do
```
Comment (IsabelleTornqvist) [inline Server/src/compile.sh]: Plz add small header for what this does

**Review by IsabelleTornqvist — CHANGES_REQUESTED** [review]

File: Application/src/net/Decode.java [hunk 2]
```diff
@@ -69,13 +67,14 @@ void execute() {
 
                 // Adds message to the DiagramView.
                 message(
+                        id,
                         DiagramView.getDiagramViewInView().getDraw(),// TODO change if you want multiple diagrams.
```
Comment (timjon) [inline Application/src/net/Decode.java]: So the feature is not done?

**Review by timjon — CHANGES_REQUESTED** [review]

File: Server/src/compile.sh [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Author: Pontus Laestadius
+# Version: 1.0
+
+# Iterates over all erlang source files in the current directory.
+for f in *.erl; do
+	
+	# Compiles the erlang file.
+	erlc -v $f
+	
+	# Exho the terminal that it compiled.
```
Comment (IsabelleTornqvist) [inline Server/src/compile.sh]: Thats not a word :D

Comment (pontuslaestadius) [reply Server/src/compile.sh]: HAHAHAHAHAHAHAHAHHAHAHAHAHHAHAHAHHAHAH
AH
AH
AHHAHAHAHAH
HAHAHAH
AHHAHAH
HAH
AH
HA
HAHAHAHHAH
AH
HAH
AH
AH HA
HAHAHAHAHH

**Review by IsabelleTornqvist — APPROVED** [review]

**Review by timjon — APPROVED** [review]

**Pull request merged**